### ISSUE TITLE
Buildall is stuck on the webletsbuilder

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldtech/builders#47]

**Comment by robvanmieghem** [comment]
```log
+ cd playground
[-] build: + yarn install
[+] build: yarn install v1.22.19
[+] build: [1/4] Resolving packages...
[+] build: [2/4] Fetching packages...
[+] build: [3/4] Linking dependencies...
[-] build: warning "grid3_client > ts-rmb-http-client > @polkadot/keyring@9.6.2" has unmet peer dependency "@polkadot/util@9.6.2".
[-] build: warning "grid3_client > ts-rmb-http-client > @polkadot/keyring@9.6.2" has unmet peer dependency "@polkadot/util-crypto@9.6.2".
[-] build: warning " > sass-loader@8.0.2" has unmet peer dependency "webpack@^4.36.0 || ^5.0.0".
[-] build: warning Workspaces can only be enabled in private projects.
[+] build: [4/4] Building fresh packages...
[-] build: warning Error running install script for optional dependency: "/app/playground/node_modules/sodium-native: Command failed.
[-] build: Exit code: 1
[-] build: Command: node-gyp-build \"node preinstall.js\" \"node postinstall.js\"
[-] build: Arguments:
[-] build: Directory: /app/playground/node_modules/sodium-native
[-] build: Output:
[-] build: libtool is required, but wasn't found on this system
[-] build: ./configure: line 5: ./configure: not found
[-] build: /app/playground/node_modules/sodium-native/preinstall.js:119
[-] build:     if (err) throw err
[-] build:              ^
[-] build:
[-] build: Error: ./configure exited with 127
[-] build:     at ChildProcess.<anonymous> (/app/playground/node_modules/sodium-native/preinstall.js:149:25)
[-] build:     at ChildProcess.emit (node:events:513:28)
[-] build:     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
[-] build:
[-] build: Node.js v18.14.2"
[+] build: info This module is OPTIONAL, you can safely ignore this error
[-] build: error /app/playground/node_modules/deasync: Command failed.
[-] build: Exit code: 1
[-] build: Command: node ./build.js
[-] build: Arguments:
[-] build: Directory: /app/playground/node_modules/deasync
[-] build: Output:
[-] build: gyp info it worked if it ends with ok
[-] build: gyp info using node-gyp@9.3.0
[-] build: gyp info using node@18.14.2 | linux | x64
[-] build: gyp ERR! find Python
[-] build: gyp ERR! find Python Python is not set from command line or npm configuration
[-] build: gyp ERR! find Python Python is not set from environment variable PYTHON
[-] build: gyp ERR! find Python checking if "python3" can be used
[-] build: gyp ERR! find Python - "python3" is not in PATH or produced an error
[-] build: gyp ERR! find Python checking if "python" can be used
[-] build: gyp ERR! find Python - "python" is not in PATH or produced an error
[-] build: gyp ERR! find Python
[-] build: gyp ERR! find Python **********************************************************
[-] build: gyp ERR! find Python You need to install the latest version of Python.
[-] build: gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
[-] build: gyp ERR! find Python you can try one of the following options:
[-] build: gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
[-] build: gyp ERR! find Python   (accepted by both node-gyp and npm)
[-] build: gyp ERR! find Python - Set the environment variable PYTHON
[-] build: gyp ERR! find Python - Set the npm configuration variable python:
[-] build: gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
[-] build: gyp ERR! find Python For more information consult the documentation at:
[-] build: gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
[-] build: gyp ERR! find Python **********************************************************
[-] build: gyp ERR! find Python
[-] build: gyp ERR! configure error
[-] build: gyp ERR! stack Error: Could not find any Python installation to use
[-] build: gyp ERR! stack     at PythonFinder.fail (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:330:47)
[-] build: gyp ERR! stack     at PythonFinder.runChecks (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:159:21)
[-] build: gyp ERR! stack     at PythonFinder.<anonymous> (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:202:16)
[-] build: gyp ERR! stack     at PythonFinder.execFileCallback (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/find-python.js:294:16)
[-] build: gyp ERR! stack     at exithandler (node:child_process:427:5)
[-] build: gyp ERR! stack     at ChildProcess.errorhandler (node:child_process:439:5)
[-] build: gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
[-] build: gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:289:12)
[-] build: gyp ERR! stack     at onErrorNT (node:internal/child_process:476:16)
[-] build: gyp ERR! stack     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[-] build: gyp ERR! System Linux 5.15.0-1002-kvm
[-] build: gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
[-] build: gyp ERR! cwd /app/playground/node_modules/deasync
[-] build: gyp ERR! node -v v18.14.2
[-] build: gyp ERR! node-gyp -v v9.3.0
[-] build: gyp ERR! not ok
[-] build: Build failed
[+] build: info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by robvanmieghem** [comment]
version v1.7.0-qa3 did not even make it this far:
```log
 build: <--- Last few GCs --->
[-] build:
[-] build: [93:0x7f8b288af2b0]   725483 ms: Mark-sweep 5046.9 (5166.8) -> 5031.6 (5167.3) MB, 4985.1 / 0.0 ms  (average mu = 0.157, current mu = 0.084) allocation failure; scavenge might not succeed
[-] build: [93:0x7f8b288af2b0]   733793 ms: Mark-sweep 5048.2 (5167.8) -> 5032.6 (5168.6) MB, 7918.6 / 0.0 ms  (average mu = 0.093, current mu = 0.047) allocation failure; scavenge might not succeed
[-] build:
[-] build:
[-] build: <--- JS stacktrace --->
[-] build:
[-] build: FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
[-] build: error Command failed with signal "SIGABRT".
[+] build: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by robvanmieghem** [comment]
solved the JavaScript heap out of memory: 3f6f529f23736d27192410fcaddc088bf1592426

**Comment by robvanmieghem** [comment]
Fixed: f182c5877a0b74aa4dc2cbe8c25353b08ec24a4f
